vMenu/MainMenu.cs

```diff
@@ -158,6 +188,13 @@ public MainMenu()
                                 SetRichPresence($"Enjoying FiveM!");
                             }
                         }
+                        else if (args[0].ToString().ToLower() == "dumplang")
```

Comment:
Please add this functionality directly on the server side. There's no need to send this huge list to the server when the server already has the file.